src/chart/radar/RadarView.ts

```diff
@@ -177,92 +177,108 @@ class RadarView extends ChartView {
             })
             .execute();
 
-        data.eachItemGraphicEl(function (itemGroup: graphic.Group, idx) {
-            const itemModel = data.getItemModel<RadarSeriesDataItemOption>(idx);
-            const polyline = itemGroup.childAt(0) as graphic.Polyline;
-            const polygon = itemGroup.childAt(1) as graphic.Polygon;
-            const symbolGroup = itemGroup.childAt(2) as graphic.Group;
-            // Radar uses the visual encoded from itemStyle.
-            const itemStyle = data.getItemVisual(idx, 'style');
-            const color = itemStyle.fill;
-
-            group.add(itemGroup);
-
-            polyline.useStyle(
-                zrUtil.defaults(
-                    itemModel.getModel('lineStyle').getLineStyle(),
-                    {
-                        fill: 'none',
-                        stroke: color
+            data.eachItemGraphicEl(function (itemGroup: graphic.Group, idx) {
```

Comment:
Please don't change the indent if there's no problem with the old code. It makes the diff hard to review.